api/candidatesApiClient.ts

```diff
@@ -1,10 +1,24 @@
-﻿export async function getCandidates(): Promise<string[]> {
-    
-    //get json candidates from site
-    
-    //get names from candidates
-    
-    //put into list
-    
-    return ["Michael Walker", "Stephen Jackson"];
+﻿export interface ListResponse<T> {
+    items: T[];
+    totalNumberOfItems: number;
+    page: number;
+    nextPage: string;
+    previousPage: string;
+}
+export interface Candidate{
+    id: number;
+    firstName: string;
+    lastName: string;
+    guid: string;
+}
+
+export async function getCandidates(): Promise<ListResponse<Candidate>> {
+    const response = await fetch(`https://testswitch-api-staging.herokuapp.com/candidates`);    
+    if(response.status==200){
```

Comment:
could you use `===` instead of `==`?

(or if you prefer, there is also `response.ok` - which will check for any 2xx response)

Reply:
replaced with response.ok